sdks/python/apache_beam/runners/interactive/interactive_runner_test.py

```diff
@@ -155,7 +154,6 @@ def process(self, element):
   @unittest.skipIf(
       sys.version_info < (3, 5, 3),
       'The tests require at least Python 3.6 to work.')
-  @timeout(60)
```

Comment:
Do you want to replace the decorators with `@pytest.mark.timeout(60)`, etc.?

Reply:
All of these timeouts are significantly less than the default. I don't see any reason these tests should be given less time to run than others.